CONTRIBUTING.rst

```diff
@@ -66,8 +66,8 @@ Ready to contribute? Here's how to set up `proxysql_tools` for local development
 
 3. Install your local copy into a virtualenv::
 
-    $ make virtualenv
-    $ source env/bin/activate
+    $ mkvirtualenv proxysql-tools
```

Comment:
Well its not really easier because you still have to run two commands. Furthermore, we require the user to install virtualenvwrapper package. At least mention that in the CONTRIBUTING doc.